saga-web/src/main/resources/saga-frontend/src/app/app.module.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
```

Comment:
@WillemJiang Can we keep these license as the file is not modified by us

Reply:
Yes, we need to keep the license there. That is why I added the comments month ago.
"I need to know if this file is generated or write by hand.
If it is write by hand we need to apply the apache License header here.
If not, we should find a right way to specify the license."

Reply:
@asifdxtreme @WillemJiang I have added ASF headers to all files written by me. 
Any files that were used from the framework are under the MIT license. 
The text has been added as per Asif's comment above to the files that we have not modified.